src/test/java/unicash/model/transaction/DateTimeTest.java

```diff
@@ -1,8 +1,10 @@
 package unicash.model.transaction;
 
```

Comment:
Now that we have three different ways to input datetimes, how shall we define equality between our DateTime objects? If we only take the `dateTime` attribute into consideration, then we could consider adding unit tests to check if dateTime objects that have been instantiated in different ways are still equal. That is:
```
DateTime d1 = new DateTime("01-01-2023 00:00");
DateTime d2 = new DateTime("2023-01-01 00:00");
assertTrue(d1.equals(d2));
```

If we also want to compare the value of `originalString`, then we will need to bake that logic into the DateTime class's `equals` method! 😄

Reply:
My understanding of this is that regardless of the input format by the user, the DateTime would be equal as long as it is the same date & time. I'll ask the team about this during our meeting, to double confirm. Thanks for the suggestion for the unit tests, i'll add that in!